docs/data/material/components/grid2/CenteredElementGrid.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 import Avatar from '@mui/material/Avatar';
 import Box from '@mui/material/Box';
-import Grid from '@mui/material/Unstable_Grid2';
+import Grid from '@mui/material/Grid2';
```

Comment:
Not related to this PR but I still see deprecated system props used in this demo. I found that there is no `Unstable_Grid` in the list of `removeSystemProps` codemod, it should be added to the list.

Reply:
I'll open a separate PR for this after we merge this one 👌🏼